imessage-exporter/src/app/runtime.rs

```diff
@@ -166,7 +166,12 @@ impl<'a> Config<'a> {
     /// let app = Config::new(options).unwrap();
     /// ```
     pub fn new(options: Options) -> Result<Config, RuntimeError> {
-        let conn = get_connection(&options.db_path).map_err(RuntimeError::DatabaseError)?;
+        let conn = if options.ios {
```

Comment:
Instead of storing `iOS` as a Boolean, we should create an enum like `ExportSource` that has variants for `MacOS` and `iOS` so we may pattern match on them later should we expand this logic in the future.

Reply:
Yes I agree, I wasn't sure how to go about it since I was largely going off of what I saw, is there a design doc or class diagram I could refer to?

Reply:
https://doc.rust-lang.org/rust-by-example/custom_types/enum.html

Reply:
Yes I understand the basic concept, what I think of though is a relatively simple implementation 
```rust
enum ExportPlatform {
    MacOS,
    IOS,
}

match export_platform {
    ExportPlatform::MacOS => {
        // macOS-specific export code
    }
    ExportPlatform::IOS => {
        // iOS-specific export code
    }
}
```

I meant where in the program would I put this enum? Like is there a specific hierarchy or structure you use with your "classes" (outdated term) that I could follow, or am I free to put in anywhere I see fit?

Reply:
That implementation looks reasonable to me. It should live somewhere in the library if it is used in the library, otherwise it should live in the binary's `options.rs` for now.